waltid-services/waltid-wallet-api/start-db.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
```

Comment:
not sure it should be part of the project
I think it might be a leftover, I think it was a convenience script when api was tested against postgres, but it's not referenced anywhere in the project
so maybe this can be removed?